tests/acceptance/test_acceptance/test_activate.py

```diff
@@ -221,16 +220,16 @@ def test_activate__feature(session_obj, feature_key, expected_response,
 ]"""
 
 
-@pytest.mark.parametrize("decision_type, expected_response, expected_status_code", [
-    ("experiment", expected_activate_type_exper, 200),
-    ("feature", expected_activate_type_feat, 200),
+@pytest.mark.parametrize("decision_type, expected_response, expected_status_code, bypass_validation", [
+    ("experiment", expected_activate_type_exper, 200, False),
```

Comment:
There are some request which won't pass validation but we need to check response. That's why added one more arg, to bypass validation for request.